### PR TITLE
Inital commit

### PR DESCRIPTION
**Pull request by anttikajanus**
@dotMorten Please review.

Added desktop demo for adding and editing geometries in 3d.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ This project contains source code for native Windows Desktop, Store, and Phone a
 * [`Using an External GPS Device`](src/ExternalNmeaGPS) - A Windows Desktop app showing how to use data from an external GPS or NMEA log file to power the MapView's LocationDisplay.
 * [`Measure Tool`](src/MeasureTool) - A sample demonstrates how to extend the editor to create a measure tool.
```
Comment (katydalton) [inline README.md]: *demonstrating

File: src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+using Windows.UI;
+#else
+using System.Windows.Media;
+#endif
+
+namespace SceneEditingDemo.Helpers
+{
+    /// <summary>
+    /// <see cref="SceneEditHelper"/> provides methods for creating and editing geometries on the SceneView.
+    /// When draw or edit is requested, draw experience is started in the assosiated SceneView that provides
```
Comment (katydalton) [inline src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs]: *associated

File: src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+		{
+			Width = 5,
+			Color = Color.FromArgb(100, 255, 255, 255),
+			Style = SimpleLineStyle.Dot
+		};
+
+		#endregion // Default draw symbols
+
+        /// <summary>
+        /// Gets the value indicating wheter there is a draw or edit session ongoing.
```
Comment (katydalton) [inline src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs]: *whether

File: src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+			}
+			finally
+			{
+				cleanup();
+			}
+			return result;
+		}
+
+        /// <summary>
+        /// Edit existing <see cref="Polyline"/>. This will activate editing experience on the map. Edit is completed on douple click.
```
Comment (katydalton) [inline src/SceneViewEdit/SceneEditingDemo/Helpers/SceneEditHelper.cs]: *double

**Comment by anttikajanus** [comment]
@dotMorten please review

**Pull request merged**